nucleus/common/internal-api/src/main/java/org/glassfish/internal/deployment/DeploymentTracing.java

```diff
@@ -83,7 +88,18 @@
 
     }
 
-    public static enum ModuleMark {
+    public enum AppStage {
+        OPENING_ARCHIVE,
+        VALIDATE_TARGET,
+        INIT_ARCHIVE_HANDLER,
+        CREATE_DEPLOY_CONTEXT,
+        DETERMINE_APP_NAME,
+        PROVIDE_APPINFO,
+        READ_DESCRIPTORS, CLEANUP, SWITCH_VERSIONS, DEPLOY, PREPARE, INITIALIZE, REGISTRATION, CLASS_SCANNING, CONTAINER_START, LOAD_METADATA, CREATE_CLASSLOADER, PREPARE_MODULES, LOAD, PREPARE_EVENTS, LOAD_EVENTS;
```

Comment:
Can we get these items put onto new lines to prevent excessive/unnecessary line length